honeybee/radiance/material/plastic.py

```diff
@@ -49,6 +49,44 @@ def __init__(self, name, r_reflectance=0, g_reflectance=0, b_reflectance=0,
            very rough surface. Roughness values greater than 0.2 are not very realistic.
            (Default: 0)."""
 
+    def to_json(self):
+        """Translate radiance material to json
+        {
+            "type": "plastic", // Material type
+            "name": "", // Material Name
+            "r_reflectance": float, // Reflectance for red
+            "g_reflectance": float, // Reflectance for green
+            "b_reflectance": float, // Reflectance for blue
+            "specularity": float, // Material specularity
+            "roughness": float // Material roughness
+        }
+        """
+        return {
+            "type": "plastic",
+            "name": self.name,
+            "r_reflectance": self.r_reflectance,
+            "g_reflectance": self.g_reflectance,
+            "b_reflectance": self.b_reflectance,
+            "specularity": self.specularity,
+            "roughness": self.roughness
+        }
+
+    @classmethod
+    def from_json(cls, rec_json):
+        """Make radiance material from json
+        {
+            "name": "", // Material Name
+            "r_reflectance": float, // reflectance for red
+            "g_reflectance": float, // reflectance for green
+            "b_reflectance": float, // reflectance for blue
+            "refraction": float, // Index of refraction
```

Comment:
similar to metal. refraction should be removed and specularity and roughness should be added.